frontend/components/DashboardAuctionsView.vue

```diff
@@ -45,6 +54,10 @@ export default Vue.extend({
             type: Array as Vue.PropType<CollateralRow[]>,
             default: () => [],
         },
+        callees: {
+            type: Object as Vue.PropType<CalleeAddresses>,
+            default: () => {},
```

Comment:
This returns undefined. If you want to return an empty object:
```suggestion
            default: () => ({}),
```